app/javascript/components/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Greeting from './Greeting';
+
+function App() {
+  return (
+    <div className="App">
+      <h2>Someone says</h2>
+      <Greeting />
+    </div>
+  );
+}
+
+export default App;
```

Comment:
- [ ] `App`component needs to have a route that leads to the `Greeting` component, with routes (from react-router)
![image](https://user-images.githubusercontent.com/62705947/223468882-dfc6e3e6-cae6-4024-a5a0-ee95756b649e.png)